hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -32,14 +32,36 @@ <h2 class="formH2">Complete your application</h2>
             {% elif application is not none and review is not none %}
                 <br />
                 {% if review.status == "Accepted" %}
-                    <h4 class="formH2"><b>Congratulations!</b> You've been accepted into {{ hackathon_name }}!</h4>
-                    <p><strong>Status</strong>: {{ status }}</p>
+                    {% if application.rsvp == True %}
+                        <h4 class="formH2"><b>Thanks for RSVP'ing!</b> See you at {{ hackathon_name }}!</h4>
+                    {% elif application.rsvp == False %}
+                        <h4 class="formH2"><b>Thanks for letting us know.</b> Hope to see you next year at {{ hackathon_name }}</h4>
+                    {% else %}
+                        <h4 class="formH2"><b>Congratulations!</b> You've been accepted into {{ hackathon_name }}!</h4>
+                    {% endif %}
+                    <p style="display: flex;">
+                        <strong>Status:&nbsp;</strong>
+                        {% if status == "Cannot Attend (Declined)" %} 
```

Comment:
To be consistent, I would keep these to be `if application.rsvp is True` and False for below. That way if the satus strings are updated in the view code, the templates don't need to be updated as well.